docker/hive/Dockerfile

```diff
@@ -83,11 +83,12 @@ RUN wget -P ${ATLAS_HOME}/hook/hive/atlas-hive-plugin-impl/ https://repo1.maven.
 ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.5.0/wait /wait
 RUN chmod +x /wait
 
+COPY init-hive.sh /
 COPY start-hive.sh /
 COPY log4j.json.properties .
 
 RUN mkdir -p $ATLAS_HOME/hook-bin/
 COPY atlas/import_hive.sh $ATLAS_HOME/hook-bin/
 RUN chmod +x /$ATLAS_HOME/hook-bin/import_hive.sh
 
-CMD /wait && /start-hive.sh
\ No newline at end of file
+CMD /wait && /init-hive.sh && /start-hive.sh
```

Comment:
why not always run init_hive?  why its part of the CMD? we can always run it before this line no?

Reply:
I don't understand. Do you want to run it within the docker build?